sdk/personalizer/Azure.AI.Personalizer/src/Models/MultiSlotClient.cs

```diff
@@ -1,7 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure.Core;
+
 namespace Azure.AI.Personalizer
 {
+    [CodeGenModel("MultiSlotClient")]
```

Comment:
The CodeGenModel decorator is for renaming a class, but you are not renaming it, so it is unnecessary